google/resource_logging_project_exclusion_test.go

```diff
@@ -0,0 +1,240 @@
+package google
+
+import (
+	"fmt"
+	"strconv"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+	"google.golang.org/api/logging/v2"
+)
+
+func TestAccLoggingProjectExclusion_basic(t *testing.T) {
+	t.Parallel()
+
+	exclusionName := "tf-test-exclusion-" + acctest.RandString(10)
+
+	var exclusion logging.LogExclusion
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckLoggingProjectExclusionDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccLoggingProjectExclusion_basic(exclusionName),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckLoggingProjectExclusionExists("google_logging_project_exclusion.basic", &exclusion),
+					testAccCheckLoggingProjectExclusion(&exclusion, "google_logging_project_exclusion.basic")),
+			},
+		},
+	})
+}
+
+func TestAccLoggingProjectExclusion_importBasic(t *testing.T) {
+	t.Parallel()
+
+	exclusionName := "tf-test-exclusion-" + acctest.RandString(10)
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			resource.TestStep{
+				Config: testAccLoggingProjectExclusion_basic(exclusionName),
+			},
+
+			resource.TestStep{
```

Comment:
Since you already have a test that sets up infrastructure with _basic, we can save a bit of time in the test suite by adding this step into the _basic test.